rainmaker_orchestrator/server.py

```
@@ -1,426 +1,628 @@
+    
+    return "\n".join(output)
+
+
+# ============================================================================
+# Alert Handling Endpoints
+# ============================================================================
+
+@app.post("/webhook/alert", tags=["Alerts"])
+async def handle_alert_webhook(
+    alert: AlertPayload,
+    background_tasks: BackgroundTasks
```

Ruff (0.14.10)

Unused function argument: `background_tasks`

(ARG001)